<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>signin_widget</class>
 <widget class="QWidget" name="signin_widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>250</width>
    <height>362</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="username_groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>231</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string>Username</string>
   </property>
   <widget class="QLineEdit" name="username_lineEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;only &lt;span style=&quot; font-weight:600;&quot;&gt;Alphabetic leters &lt;/span&gt;and &lt;span style=&quot; font-weight:600;&quot;&gt;Numbers&lt;/span&gt; and &lt;span style=&quot; font-weight:600;&quot;&gt;_ character&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">border-left-color: #fff;
border-left-style: solid;
border-left-width: 5px;
text-indent: 3px;
padding-left:3px;
</string>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
    <property name="placeholderText">
     <string>Enter Your User Name</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="password_groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>231</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string>Password</string>
   </property>
   <widget class="QLineEdit" name="password_lineEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="acceptDrops">
     <bool>true</bool>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;only &lt;span style=&quot; font-weight:600;&quot;&gt;Alphabetic leters &lt;/span&gt;and &lt;span style=&quot; font-weight:600;&quot;&gt;Numbers.&lt;/span&gt; you must use exactly one of theas characters [&lt;span style=&quot; font-weight:600;&quot;&gt;$&lt;/span&gt;, &lt;span style=&quot; font-weight:600;&quot;&gt;%&lt;/span&gt;]&lt;/p&gt;&lt;p&gt;* 5 &amp;lt; len &amp;lt; 10&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">border-left-color: #fff;
border-left-style: solid;
border-left-width: 5px;
padding-left:3px;</string>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>Enter Your Password</string>
    </property>
    <property name="clearButtonEnabled">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="visible_OF">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>20</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	background-color: transparent;
	color: #fff;
	font-weight: bold;
	border-style: solid;
	border-width: 1px;
	border-radius: 17px;
	border-color: transparent;
	padding: 10px;

}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../res/icon/visibility-off.png</normaloff>../res/icon/visibility-off.png</iconset>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="login_pushButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	background-color: #003bbd;
	color: rgb(255, 255, 255);
	font-weight: bold;
	border-style: solid;
	border-width: 1px;
	border-radius: 5px;
	padding: 10px;
	font: 700 9pt &quot;Exo 2&quot;;
}
</string>
   </property>
   <property name="text">
    <string>LOGIN</string>
   </property>
  </widget>
  <widget class="QLabel" name="goto_signup_link">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>310</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Exo 2</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget
{
	color: rgb(118, 118, 118);
  	font: 400 10pt &quot;Exo 2&quot;;
}
</string>
   </property>
   <property name="text">
    <string>Don't have account?</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>249</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget
{
	color: #263238;
  	font: 900 14pt &quot;Exo 2&quot;;
}</string>
   </property>
   <property name="text">
    <string>Welcome Back</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
